releases/splinter_0_4_release_overview.md

```diff
@@ -0,0 +1,236 @@
+# Splinter v0.4 Release Overview
+
+Splinter v0.4 is a major new release. Here's a summary of the new and noteworthy
+features in this release. For detailed changes related to the v0.4 release
+(which is the same as the v0.3.19 release), see the [Splinter release
```

Comment:
I think we should remove the 0.3.x reference here, since its only an interesting comment sort-term.

Reply:
How will people know that 0.4 is the same as 0.3.19?  Confusion is possible, especially if there aren't any v0.4 release notes.

Reply:
Deleted this phrase. We can add this information to the release notes (per an off-line discussion).